### PR TITLE
Update orvibo-Moisture-sensor.groovy

### PR DESCRIPTION
**Pull request by weihuan1111**
update thirdreality battery percent report Update

### CODE REVIEW

**Comment by weihuan1111** [comment]
@greens please help to review the merge require,thanks

**Review by greens — CHANGES_REQUESTED** [review]
fix string comparison

**Comment by weihuan1111** [comment]
@greens hello, i have update the code and fix comparison, please help to review the merge require,thanks

**Review by weihuan1111 — COMMENTED** [review]
update orvibo-moisture-sensor.groovy

File: devicetypes/smartthings/orvibo-Moisture-Sensor.src/orvibo-Moisture-Sensor.groovy [hunk 1]
```diff
@@ -156,11 +157,13 @@ def getBatteryPercentageResult(rawValue) {
 	log.debug "Battery Percentage"
 	def result = [:]
 	def manufacturer = getDataValue("manufacturer")
+	def application = getDataValue("application")
+	int application_thirdreality = application.toInteger()
```
Comment (greens) [inline devicetypes/smartthings/orvibo-Moisture-Sensor.src/orvibo-Moisture-Sensor.groovy]: don't use two variables for this, just cast immediately. `getDataValue().toInteger()`

**Review by weihuan1111 — COMMENTED** [review]
update

**Comment by weihuan1111** [comment]
@greens hello, i have update the code, please help to review the merge require,thanks

**Review by greens — APPROVED** [review]

**Pull request merged**